boards/arm/mimxrt1010_evk/mimxrt1010_evk_defconfig

```diff
@@ -12,3 +12,5 @@ CONFIG_SERIAL=y
 CONFIG_GPIO=y
 CONFIG_ARM_MPU=y
 CONFIG_HW_STACK_PROTECTION=y
+CONFIG_PINMUX=y
+CONFIG_PINMUX_MCUX_RT=y
```

Comment:
Can you update Kconfig here to just set `CONFIG_PINMUX=y` and drop `CONFIG_PINMUX_MCUX_RT=y` from all the defconfigs.

Add something in the SoC like we do for CONFIG_GPIO.

Reply:
update.